design/specs/table-import.md

```diff
@@ -14,153 +14,175 @@ Adding a new table from a file import allows users to input data into Mathesar w
 While data file import is a baseline feature for most applications in this category, additional functionality will be implemented in the future to enhance the data import process, such as automatic type detection, data cleansing, error detection, and others.
 
 # User Experience
-## Impact
-This feature will improve the user experience for users who need to input data into Mathesar by reducing the required effort when populating tables.
 
-This feature will also support the experience of first-time users, as it is expected by users of similar applications and would cause dissatisfaction if missing.
+## Navigating to the Data Import page
 
-## User Flow
+![](https://share.balsamiq.com/c/p3dqSXr66qa6EnPyykqpyZ.png)
 
-![](/assets/design/specs/table-import/Z3ngzR9.png)
+Data import navigation link will be found on the active schema page. Since we are only allowing users to import tables for now, positioning it with the existing tables/ active schema view made sense. 
 
+## Importing data into a new table
 
-### User Adds a Table
-The user starts the process of adding a table by clicking on any of the available options. The user can add a table from the sidebar menu(1) or the 'New' menu located on the top navigation bar in the main content area.
+![](https://share.balsamiq.com/c/jyszUUBfSvoCqd6qWQcz3G.png)
 
-![](/assets/design/specs/table-import/DH3XRuj.png =400x)
+### Defining the New Table
 
-#### Notes
-- We can implement the 'New' menu at a later iteration.
+When the user jumps to the upload view. They see a name input which can be added or not added. If not added by user the table will be given a default name and that can also be changed after the import is confirmed.
```

Comment:
I have discussed this with Pavish and the current backend won't handle renaming the table before it is created. We will have to move table rename to the confirmation step instead.

Reply:
I did not get this, could we discuss this please?